File: w09/d02/homework/brainteasers/letter_counter.rb
alphabet = {}
i = 1

("a".."z").each { |letter| 
	alphabet[letter] = i
	i+=1 }

puts alphabet

def countr(string, alpha)
	total = 0
	string.split("").each do |n|
		if n != " "
		num = alpha[n.downcase]
		total+=num
		end
	end
	return total
end

puts countr("Dog", alphabet)

File: w09/d02/homework/brainteasers/letter_counter.js
var alphabet = {}

var string = "abcdefghijklmnopqrstuvwxyz";
var alpha_array = string.split("");
var i = 1;

alpha_array.forEach(function(each){
	alphabet[each] = i;
	i++;
})

console.log(alphabet);

var countr = function(string, alpha){
	var total = 0;
	string.split('').forEach(function(n){
		if(n != " "){
			var num = alpha[n.toLowerCase()];
			total += num
		}
	})
	return total
}

console.log(countr("dog", alphabet))

